packages/components/Organisms/Grants/grants-status.js

```diff
@@ -0,0 +1,68 @@
+import moment from 'moment'
+import formatdateMixin from '@dpc-sdp/ripple-global/mixins/formatdate'
+const formatDate = formatdateMixin.methods.formatDate
+
+export const defaultStatusTerms = {
+  open: 'Open',
+  closed: 'Closed',
+  ongoing: 'Ongoing',
+  openingSoon: (startdate) => `Opening on ${startdate}`,
+  closingSoon: (end, now) => {
+    const daysRemaining = parseInt(end.diff(now, 'days'))
+    if (daysRemaining > 1) {
+      return `Open, closing in ${daysRemaining} days`
+    } else if (daysRemaining === 1) {
+      return `Open, closing in ${daysRemaining} day`
+    }
+    return `Open, closing today`
+  }
+}
+export default function calcStatus (startDate, endDate, displaySoon = true, statusTerms = defaultStatusTerms) {
```

Comment:
This should be used in https://github.com/dpc-sdp/ripple/blob/9ea0755662046d305c9f450c1b4861a01f535a28/packages/components/Organisms/Grants/GrantsOverview.vue#L71 - but i didn't want to touch the implementation of this just yet. I think it needs a mixin, but that should be a follow up PR.